.ci/manywheel/build_common.sh

```diff
@@ -253,11 +253,11 @@ make_wheel_record() {
     FPATH=$1
     if echo $FPATH | grep RECORD >/dev/null 2>&1; then
         # if the RECORD file, then
-        echo "$FPATH,,"
+        echo "\"$FPATH\",,"
     else
         HASH=$(openssl dgst -sha256 -binary $FPATH | openssl base64 | sed -e 's/+/-/g' | sed -e 's/\//_/g' | sed -e 's/=//g')
         FSIZE=$(ls -nl $FPATH | awk '{print $5}')
-        echo "$FPATH,sha256=$HASH,$FSIZE"
+        echo "\"$FPATH\",sha256=$HASH,$FSIZE"
```

Comment:
@jithunnair-amd @malfet 
`RECORD` is a CSV file (Excel dialect), and file names need to be quoted when it contains comma.

We should use csv module in python to maximize compatibility, but for file names without quotes this is good enough.

Reply:
#funkybug ... thanks!

Reply:
![image](https://github.com/user-attachments/assets/b419fdd7-e624-424c-840c-2512bf9882ac)
Confirm the problem is fixed